pre-commit/action.yaml

```diff
@@ -18,10 +21,20 @@ runs:
       with:
         token: ${{ inputs.token }}
 
+    - name: Set option
+      id: set-option
+      run: |
+        if [ -z "${{ inputs.base-branch }}" ]; then
+          echo "option=--all-files" >> $GITHUB_OUTPUT
+          exit 0
```

Comment:
Is this for debugging?

```suggestion
```

Reply:
Without this line, the `from-ref` option is specified when the `base-branch` is empty.

Reply:
Ah, I see. It's a kind of early-return.
Then it's okay. I wasn't used to this style in GitHub Actions.